thesis/02-functionality.tex

```diff
@@ -1,27 +1,28 @@
 \subsection{System actors}
 
-Based on the requirements analysis, we have  two types of actors in our system:
+Based on the requirements analysis, we have determined two types of actors in our system:
 
 \begin{description}
 	\item[Folder] \label{actors:folder} - A person who carries out the most important functionality in our system,
-		that is - folds the origami, based on the provided simulation. This person does not need to have any
-		knowledge about origami creation nor the simulation process.
+		that is - folds the origami figure following the provided interactive instruction.
```

Comment:
the origami figure  following ... -> origami figures following provided interactive instructions.

We have not defined any particular origami figure in the past.